arch/arm/neon_intrins.h

```diff
@@ -25,8 +25,8 @@
     out.val[3] = vqsubq_u16(a.val[3], b); \
 } while (0)
 
-#  if defined(__clang__) && defined(__arm__) && defined(__ANDROID__)
-/* Clang for 32-bit Android has too strict alignment requirement (:256) for x4 NEON intrinsics */
+#  if defined(__clang__) && defined(__arm__) && (defined(__ANDROID__) || defined(__APPLE__))
+/* Clang for 32-bit Android and iOS has too strict alignment requirement (:256) for x4 NEON intrinsics */
```

Comment:
Maybe use ``__ENVIRONMENT_IPHONE_OS_VERSION_MIN_REQUIRED__`` instead of ``__APPLE__``, because  ``__APPLE__`` is defined on macOS too.

Reply:
`__arm__` being defined is also required, so I don't think it's an issue since there is no 32 bit arm macOS.